### ISSUE TITLE
Feature Request: Encrypted SeedQR

### ISSUE THREAD

**Issue by ViniSug** [Blockstream/Jade#87]
It would be very interesting for security to implement an encrypted SeedQR similar to the one used in the Krux project.
https://github.com/odudex/krux

**Comment by JamieDriver** [comment]
Indeed.  We can look at adding support for such scanning features if there is sufficient user demand.

**Comment by odudex** [comment]
It would be really cool to have it in Jade! At the Krux side we would do all it's possible to make the QRs interoperable between the devices!
[Here](https://github.com/odudex/krux/blob/develop_rc/docs/encrypted-qr-codes.en.md) is a brief explanation of how we did it in Krux.
